lesson_10/test_tasks.py

```diff
@@ -0,0 +1,82 @@
+"""Module test tasks"""
+
+
+from homework_10 import log_event
+import logging
+from datetime import date
+from venv import logger
+
+"""Check if two arguments were provided"""
+
+def test_two_strings_provided():
+
+    try:
+        func_loging = log_event("tester")
+    except TypeError as miss_type:
+        print("Missing positional argument")
+
+
+#test_two_strings_provided()
+
+#Test Data
+name = "test"
+name_1 = "test1"
+status_s = "success"
+status_e = "expired"
+status_f = "failed"
+
+info = logging.info("20")
+warn = logging.warning("30")
+err = logging.error("40")
+
+logger_data = open('login_system.log', 'r')
+data = logger_data.read()
+
+"""Could find only one level of logs"""
+
+def test_logger_level():
```

Comment:
This function and other function below have no log_event call
It means that we test "nothing".
Each test should have some log_event call with certain parameters and then test/check what was changed added etc